tests/unit/lms/services/lti_outcomes_test.py

```diff
@@ -97,32 +104,37 @@ def test_record_result_requires_dict_result(self, svc, hook_result):
                 self.GRADING_ID, pre_record_hook=lambda *args, **kwargs: hook_result
             )
 
-    def test_it_signs_request_with_oauth1(self, svc, requests, oauth1_service):
-        requests.post.side_effect = OSError()
+    def test_it_signs_request_with_oauth1(self, svc, http_service, oauth1_service):
+        http_service.request.side_effect = OSError()
```

Comment:
I'm not sure why the original test was configuring `requests.post()` to raise `OSError` here, I don't think `requests.post` ever raises that. It seems like the test just wanted the `requests.post()` call to crash because it doesn't need the code after that call to execute because all it wants to do is assert about what `requests.post()` call was made :man_shrugging: I decided not to dig into it, and to just leave it the same